cmd/kube-proxy/app/server.go

```diff
@@ -774,6 +779,50 @@ func tryIPTablesProxy(iptver iptables.IPTablesVersioner, kcompat iptables.Kernel
 	return proxyModeUserspace
 }
 
+func (s *ProxyServer) reportVersion() {
+	nodeName := s.NodeRef.Name
+	done := make(chan struct{})
+	go wait.Until(func() {
+		oldNode, err := s.CoreClient.Nodes().Get(nodeName, metav1.GetOptions{})
+		if err != nil {
+			glog.Warningf("Failed to retrieve node info: %v", err)
```

Comment:
We may get spam logs in some case. See the race between kube-proxy start up and node registration #37414.

So I suggest the way #47262 did.